rocAL/rocAL/source/meta_node_resize.cpp

```diff
@@ -41,29 +41,29 @@ void ResizeMetaNode::update_parameters(MetaDataBatch* input_meta_data)
     _dst_height = _node->get_dst_height();
     for(int i = 0; i < _batch_size; i++)
     {
-        _dst_to_src_width_ratio = _dst_width / float(_src_width_val[i]);
-        _dst_to_src_height_ratio = _dst_height / float(_src_height_val[i]);
         auto bb_count = input_meta_data->get_bb_labels_batch()[i].size();
         int labels_buf[bb_count];
         float coords_buf[bb_count*4];
         memcpy(labels_buf, input_meta_data->get_bb_labels_batch()[i].data(),  sizeof(int)*bb_count);
         memcpy(coords_buf, input_meta_data->get_bb_cords_batch()[i].data(), input_meta_data->get_bb_cords_batch()[i].size() * sizeof(BoundingBoxCord));
         BoundingBoxCords bb_coords;
         BoundingBoxCord temp_box;
-        temp_box.x = temp_box.y = temp_box.w = temp_box.h = 0;
+        temp_box.l = temp_box.t= 0;
+        temp_box.r = temp_box.b = 1;
         BoundingBoxLabels bb_labels;
         for(uint j = 0, m = 0; j < bb_count; j++)
         {
             BoundingBoxCord box;
-            box.x = (coords_buf[m++] * _dst_to_src_width_ratio);
-            box.y = (coords_buf[m++] * _dst_to_src_height_ratio);
-            box.w = (coords_buf[m++] * _dst_to_src_width_ratio);
-            box.h = (coords_buf[m++] * _dst_to_src_height_ratio);
+            box.l = (coords_buf[m++] );
+            box.t = (coords_buf[m++] );
+            box.r = (coords_buf[m++] );
+            box.b = (coords_buf[m++] );
             bb_coords.push_back(box);
             bb_labels.push_back(labels_buf[j]);
         }
         if(bb_coords.size() == 0)
```

Comment:
This should never be true. Please remove

Reply:
Removed